ts/src/pro/bitmex.ts

```diff
@@ -78,6 +78,41 @@ export default class bitmex extends bitmexRest {
         return await this.watch (url, messageHash, this.extend (request, params), messageHash);
     }
 
+    async watchTickers (symbols: Strings = undefined, params = {}): Promise<Tickers> {
+        /**
+         * @method
+         * @name bitmex#watchTickers
+         * @description watches a price ticker, a statistical calculation with the information calculated over the past 24 hours for all markets of a specific list
+         * @param {string[]} symbols unified symbol of the market to fetch the ticker for
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/#/?id=ticker-structure}
+         */
+        await this.loadMarkets ();
+        symbols = this.marketSymbols (symbols, undefined, true);
+        const name = 'instrument';
+        const url = this.urls['api']['ws'];
+        const messageHashes = [];
+        if (symbols !== undefined) {
+            for (let i = 0; i < symbols.length; i++) {
+                const symbol = symbols[i];
+                const market = this.market (symbol);
+                const hash = name + ':' + market['id'];
+                messageHashes.push (hash);
+            }
+        } else {
+            messageHashes.push (name);
+        }
+        const request = {
+            'op': 'subscribe',
+            'args': messageHashes,
+        };
+        const tickers = await this.watchMultiple (url, messageHashes, this.extend (request, params), messageHashes);
+        if (this.newUpdates) {
+            return tickers;
```

Comment:
this return type is wrong

Reply:
wait, when subscribing to all instruments, it was correctly returning.

Reply:
probably i missed to check with input symbols array. i'll get to that, don't spend time on it, i missed that nuance.

Reply:
np I fixed it already